### ISSUE TITLE
Increase Color Contrast

### ISSUE THREAD

**Issue by bruce-usab** [atbcb/usab-uswds#173]
Color contrast on header text of homepage should be 7:1 or better, currently it is 6.1:1.

We should try and stick with [USWDS color choices](https://designsystem.digital.gov/utilities/color/).

I am finding x00BDE3 for the bright blue (.text-accent-cool) and x162E51 (.bg-primary-darker) for the navy background.

For reference:
- Yellow on this navy background is 12.7:1.
- Cyan on this navy background is 10.8.1.
- .text-accent-cool-light (x97D4EA) on .bg-primary-darker is 8.4:1

Font color for secondary menu is also too light.  It should be 7:1 or better.

I am getting 71767E for the gray, and 4.6:1 for the contrast.

For reference:
- 505050 against white is 8.1:1
- 575757 against whiet is 7.2:1
- 5B5B5B against white is 6.8:1
